### PR TITLE
net-analyzer/nikto: new package

### PR DESCRIPTION
**Pull request by CaseOf**
Doing pull request again since previous https://github.com/gentoo/gentoo/pull/17206 was closed.

cc @thesamesam @juippis

### CODE REVIEW

File: net-analyzer/nikto/nikto-2.1.6_p20200901.ebuild [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+# nikto provie its own libwhisker, do no use net-libs/libwhisker[ssl]
+# https://bugs.gentoo.org/533900
+RDEPEND="
+	dev-lang/perl
+	dev-perl/Net-SSLeay
+	net-analyzer/nmap
+	virtual/perl-JSON-PP
+"
+DEPEND=""
```
Comment (thesamesam) [inline net-analyzer/nikto/nikto-2.1.6_p20200901.ebuild:25]: We can drop this empty line, see [here](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-620007.3).

File: net-analyzer/nikto/nikto-2.1.6_p20200901.ebuild [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+COMMIT="fb2655b8316097723d4977c792817a21117b2736"
+MY_P="${PN}-${COMMIT}"
+SRC_URI="https://github.com/sullo/nikto/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86 ~amd64-linux ~x86-linux ~x86-macos"
+IUSE=""
+
+# nikto provie its own libwhisker, do no use net-libs/libwhisker[ssl]
```
Comment (thesamesam) [inline net-analyzer/nikto/nikto-2.1.6_p20200901.ebuild:17]: typo

File: net-analyzer/nikto/nikto-2.1.6_p20200901.ebuild [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+DESCRIPTION="Web server vulnerability scanner"
+HOMEPAGE="http://www.cirt.net/Nikto2"
+COMMIT="fb2655b8316097723d4977c792817a21117b2736"
+MY_P="${PN}-${COMMIT}"
+SRC_URI="https://github.com/sullo/nikto/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86 ~amd64-linux ~x86-linux ~x86-macos"
+IUSE=""
```
Comment (thesamesam) [inline net-analyzer/nikto/nikto-2.1.6_p20200901.ebuild:15]: drop

**Comment by CaseOf** [comment]
@thesamesam fixed

File: net-analyzer/nikto/nikto-2.1.6_p20200901.ebuild [hunk 4]
```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Web server vulnerability scanner"
+HOMEPAGE="http://www.cirt.net/Nikto2"
```
Comment (jsmolic) [inline net-analyzer/nikto/nikto-2.1.6_p20200901.ebuild:7]: You can use `https://` ;)

**Comment by CaseOf** [comment]
fixed link to https.
also added proxy-maint in metadata.xml

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**